docs/tutorial/using-native-node-modules.md

```diff
@@ -67,6 +67,16 @@ The `HOME=~/.electron-gyp` changes where to find development headers. The
 where to download the headers. The `--arch=x64` says the module is built for
 64bit system.
 
+### Manually building for a custom build of Electron
+
+To compile native Node addons against a custom build of Electron that doesn't
+match a public release, instruct `npm` to use the version of Node you have bundled
```

Comment:
To clarify, this is the path to the _source code_ (specifically, the headers) of the version of Node you built your custom build with. The build itself doesn't bundle these headers.